mpa/seg/exporter.py

```diff
@@ -45,7 +45,10 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
                 model = model.cuda(cfg.gpu_ids[0])
             else:
                 model = model.cpu()
-            export_model(model, cfg, output_path, target='openvino', output_logits=True, input_format='bgr')
+            precision = kwargs.pop('precision', 'FP32')
+            logger.info(f'Model will be exported with precision {precision}')
+
+            export_model(model, cfg, output_path, target='openvino', output_logits=True, input_format='bgr', precision=precision)
```

Comment:
We may be careful for BGR / RGB.
I see that only seg is using this specific parameter.